cmd/catalyst-node/catalyst-node.go

```diff
@@ -334,9 +314,34 @@ func main() {
 		}
 	}()
 
+	// Everything past this is booting up Serf
+	tmpFile, err := writeSerfConfig(&serfConfig)
```

Comment:
Do we need to write the config into the file? Can't serf read the config directly from string or config flags?

Reply:
Config file seemed expedient, especially when combined with the configuration `agent.Config` struct I'm using from the library. We'll change this up when we're creating a serf instance directly, of course